utils/k8s/deprovision_deployment.yml

```diff
@@ -114,22 +137,25 @@
         - block:
 
             - name: Check if the target pod is deleted.
-              shell: >
-                  kubectl get pods -l openebs.io/target=cstor-target  -n {{ operator_ns }} --no-headers
-              args:
-                executable: /bin/bash
+              k8s_facts:
+                kind: Pod
+                label_selectors:
+                  - "openebs.io/target=cstor-target"
+                  - "{{ pvname }}"
```

Comment:
How are we passing just pvname as selector? It has to be a key:value form right?